keep-ui/app/providers/providers-tiles.tsx

```diff
@@ -112,8 +114,9 @@ const ProvidersTiles = ({
   return (
     <div>
       <Title className="mb-2.5">
-        {installedProvidersMode ? "Installed Providers" : "Available Providers"}
+        {installedProvidersMode ? "Installed Providers" : linkedProvidersMode ? "Linked Providers" : "Available Providers"}
```

Comment:
Perhaps also add a small question mark icon that explains what are Linked Providers?